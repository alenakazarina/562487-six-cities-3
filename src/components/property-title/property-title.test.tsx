import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PropertyTitle from './property-title';

const CHILDREN = <div>Childrens content</div>;
const TITLE = `Property Title`;

describe(`PropertyTitle`, () => {
  it(`should render PropertyTitle correctly`, () => {
    const tree = renderer.create(
        <PropertyTitle title={TITLE} >
          {CHILDREN}
        </PropertyTitle>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
