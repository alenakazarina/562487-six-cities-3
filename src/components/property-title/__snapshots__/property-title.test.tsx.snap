// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyTitle should render PropertyTitle correctly 1`] = `
<div
  className="property__name-wrapper"
>
  <h1
    className="property__name"
  >
    Property Title
  </h1>
  <div>
    Childrens content
  </div>
</div>
`;
